api/lib/api/performance_manager/static_feed_info.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Api.PerformanceManager.StaticFeedInfo do
+  use Ecto.Schema
+
+  @primary_key {:pk_id, :id, autogenerate: true}
+  schema "static_feed_info" do
+    field(:feed_start_date, :integer)
+    field(:feed_end_date, :integer)
+    field(:feed_version, :string)
+    field(:timestamp, :integer)
+    field(:created_on, :utc_datetime)
```

Comment:
question: what's the difference between `timestamp` and `created_on`? I see most tables have a `timestamp` but not all of them have `created_on`.

Reply:
the created on is the server time when the feed info row was created. the timestamp comes from the static schedule itself that we use as the foreign key to join all of the records across static tables together, as the tables will hold more than one static schedule at a time.

we use the timestamp as the foreign key for vehicle events, trips, and event metrics as well, to tie it to a static schedule.

Reply:
If the `timestamp` is the foreign key for other tables, should that be the `pk_id` here?

Reply:
We need to rename these `timestamp` columns, they cause a lot of confusion and don't really represent what we're doing with the field.